web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -203,71 +334,12 @@ export function SelectResource({ onSelect }: SelectResourceProps) {
   );
 }
 
-const ClearSearch = ({ onClick }: { onClick(): void }) => {
-  return (
-    <Flex
-      width="120px"
-      onClick={onClick}
-      alignItems="center"
-      mt={1}
-      css={`
-        font-size: 12px;
-        opacity: 0.7;
-
-        &:hover {
-          cursor: pointer;
-          opacity: 1;
-        }
-      `}
-    >
-      <Box
-        mr={1}
-        ml={1}
-        width="18px"
-        height="18px"
-        borderRadius="4px"
-        textAlign="center"
-        css={`
-          background: ${props => props.theme.colors.error.main};
-        `}
-      >
-        <Icons.Cross size="small" />
-      </Box>
-      <Text>Clear search</Text>
-    </Flex>
-  );
-};
-
-const Grid = styled.div`
+const Grid = styled.div<{ pinTile?: boolean }>`
   display: grid;
-  grid-template-columns: repeat(auto-fill, 320px);
+  grid-template-columns: repeat(
+    auto-fill,
+    ${p => (p.pinTile ? '250.4px' : '320px')}
```

Comment:
Do we need this `0.4px`?

Reply:
lol, i copy pasta from figma. i can remove it if you like, i'm sure no one will notice the 0.4 😅 

<img width="248" alt="image" src="https://github.com/user-attachments/assets/f0fd9928-f331-4f96-8858-7a931c13a02f" />